### DISCUSSION TITLE
Dev, Don't give up💪

### DISCUSSION THREAD

**Discussion by GameMasterX0** [skyline-emu/skyline#1453] [General]
![Screenshot_2022-06-03-05-14-24-372_com android chrome](https://user-images.githubusercontent.com/106778000/171739720-a178056a-0241-4003-8269-7c1ebad60468.jpg)
Lately I've seen the progress of devs who are having a hard time. Don't give up devs, we are sure you will do your best. But do any of you want to help?

**Comment by Joemama21345** [comment]
> ![Screenshot_2022-06-03-05-14-24-372_com android chrome](https://user-images.githubusercontent.com/106778000/171739720-a178056a-0241-4003-8269-7c1ebad60468.jpg) Lately I've seen the progress of devs who are having a hard time. Don't give up devs, we are sure you will do your best. But do any of you want to help?

They did it, we finally have a update thanks to the team
